SocialCareCaseViewerApi/V1/Exceptions/CustomExceptions.cs

```diff
@@ -63,6 +63,11 @@ public class PersonNotFoundException : Exception
         public PersonNotFoundException(string message) : base(message) { }
     }
 
+    public class RelationshipNotFoundException : Exception
```

Comment:
What do you think about renaming this to `PersonalRelationshipNotFoundException` as we have other `PersonalRelationship` exceptions below.